coalib/coala_json.py

```diff
@@ -28,12 +31,38 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
-    results, exitcode, _ = run_coala(log_printer=log_printer, autoapply=False)
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            local_bears, global_bears = get_filtered_bears(
+                args.filter_by_language, log_printer)
 
-    retval = {"results": results}
+            bears = inverse_dicts(local_bears, global_bears)
+            results = []
+            for bear, sections in sorted(bears.items(),
+                                         key=lambda bear_tuple:
+                                         bear_tuple[0].name):
+                metadata = bear.get_metadata()
+                results.append({"Name": bear.name,
+                                "Description": metadata.desc,
+                                "Supported languages": bear.LANGUAGES,
```

Comment:
bears have more attributes by now, can't you just append the bear and let the JSONEncoder do the job? It should do it just fine extracting also all the other info.